features/support/hooks.rb

```diff
@@ -97,7 +97,7 @@
 end
 
 AfterStep '@javascript', '@ajax' do
-  step 'wait for ajax'
+  wait_for_requests
```

Comment:
I do not think it is wise to use the middleware everytime for `@javascript`
That will slow the tests as it needs to memoize the requests done

Reply:
@hallelujah This is not every time for `@javascript` but for the `@ajax` tag.

The problem before this commit was that many cukes where failing because it was waiting for an ajax that was never being called (for the dashboard) so it was waiting forever... now it waits only if there are requests being done.

Although I would agree that adding the wait in those specific steps that need it instead of the whole scenario would be much faster, but then imo that would mean removing the `@ajax` tag completely and adapting it to that and that would be out of scope of this PR (because I don't think that this PR makes the current situation worse).

Reply:
UPDATE: @hallelujah told me this does it for both tags, not for the combination. I wanted it only for `@ajax`, sorry 👍